pkg/errors/errors.go

```diff
@@ -48,6 +48,9 @@ var (
 	// ErrNotLogged is raised when we can't get the user token
 	ErrNotLogged = "your token is invalid. Please run 'okteto context use %s' and try again"
 
+	// ErrCtxNotSet is raised when we don't have ctx set
+	ErrCtxNotSet = fmt.Errorf("your context is not set. Please run 'okteto context use' and and select your context")
```

Comment:
```suggestion
	ErrCtxNotSet = fmt.Errorf("your context is not set. Please run 'okteto context' and and select your context")
```

Reply:
There's a double `and and` in the suggestion.